app/models/searchable.rb

```diff
@@ -150,6 +150,6 @@ def search_index
   end
 
   def can_index_in_search?
-    self.class.searchable_instances.find_by(id: self.id).present? && Whitehall.searchable_classes.include?(self.class)
+    self.class.searchable_instances.find_by(id: self.id).present? && RummagerPresenters.searchable_classes.include?(self.class)
```

Comment:
feels like `RummagerPresenters.searchable_classes.include?(self.class)` is repeated on L134, am I wrong?